requirements/requirements.txt

```diff
@@ -10,4 +10,6 @@ timezonefinder
 requests_cache
 ovos-backend-client~=0.0, >=0.0.6a10
 oauthlib~=3.0
-Flask-SQLAlchemy
\ No newline at end of file
+Flask-SQLAlchemy
+Flask-MySQLdb
```

Comment:
I think this one is not needed? in chatterbox we also used MySQL and i think did not require this?

Reply:
I am pretty sure that i hung up my docker initially cause this was missing